composer.json

```diff
@@ -1,33 +1,36 @@
 {
   "name": "treehouselabs/standards",
   "description": "Development standards by and for TreeHouse employees",
+  "license": "MIT",
   "authors": [
     {
       "name": "Peter Kruithof",
       "email": "peter@treehouse.nl"
-    },    {
+    },
+    {
       "name": "Jeroen Fiege",
       "email": "jeroen@treehouse.nl"
     }
   ],
```

Comment:
hm might be nice to get rid of the specific authors while you're at it, it's hard to maintain and besides we all work on it now and then (better to use TreeHouse development or something?)

Reply:
True, I'll update it to dev@treehouse 👍